subtrack.MAUI/Pages/SubscriptionDetails.razor

```diff
@@ -138,4 +144,18 @@ else
         await subscriptionService.Delete(Id);
         navigationManager.NavigateTo("/", replace: true);
     }
+    private string GetOccurance()
+    {
+        StringBuilder sb = new StringBuilder();
+        sb.Append("every ");
+        if(_subscription.BillingInterval==1)
+        {
+            sb.Append(_subscription.BillingOccurrence.Humanize().ToLower());
+            return sb.ToString();
+        }
+        sb.Append(_subscription.BillingInterval.ToWords());
```

Comment:
Nicely done with stringbuilder. Just an FYI you can chain the .Append calls like this 
```
return        sb.Append(_subscription.BillingInterval.ToWords())
.Append(" ")
.Append(_subscription.BillingOccurrence.Humanize().ToLower().Pluralize())
.ToString();
```